### ISSUE TITLE
heads-up about .vimrc configured options

### ISSUE THREAD

**Issue by azadbolour** [begriffs/haskell-vim-now#127]
After a successful installation, I started vim and was surprised to see a substantially different look and feel and operational mode. For example:

:set noai did not change mode to no auto indent

I then tried to vim .vimrc to see what was going on, but was surprised to find that the default presentation has the various sections of the file collapsed, and I did not know immediately how to expand them.

I have been using vim in a pretty basic non-gui manner. I guess this will force me to upgrade my vim skills. So I now know that collapse is called fold in vim.

A couple of other things I have not yet figured out:

gqap in normal mode formats a paragraph - with the new settings it seems to change the paragraph to a single line

copying with with the normal MAC mouse actions (on an iterm2 terminal)

Some hints as to what one should expect from the resulting .vimrc, and where to go to get more information about some of the important differences in modes of operation one might find could help remove the element of surprise.

Thanks.

Azad

**Comment by azadbolour** [comment]
FYI. Since I am a basic vi user ad not used to the vim options not compatible with old vi used for Haskell work, I ended up needing the ability to go back and forth between my basic settings and the new haskell-vim-now settings. For now my best solution is a couple of aliases that change the symbolic links of .vim and .vimrc:

alias tohaskellvim="ln -fsh $HOME/.config/haskell-vim-now/.vim $HOME/.vim; ln -fs $HOME/.config/haskell-vim-now/.vimrc $HOME/.vimrc"
alias tomyvim="ln -fsh $HOME/.config/my-vim/.vim $HOME/.vim; ln -fs $HOME/.config/my-vim/.vimrc $HOME/.vimrc"

Of course, I first had to copy my saved settings from the haskell-vim-now backup directory to my $HOME/.config/my-vim directory.

Azad

**Comment by begriffs** [comment]
Did our pairing session answer your questions on this issue?

**Comment by azadbolour** [comment]
I am good with the new vim now. But for people like me who might get
surprised by the different look and feel of the haskell vim, it would be
useful to give them an easy way to go back and forth between the haskell
vim and their old vim setup. I tried to do that by having aliases that
changed the symbolic links for .vimrc and .vim. But as you saw in our
pairing session, there was then an issue with the haskell vim not finding
various plugins when I went back and forth by using my aliases.

Since I am getting familiar with the new vim, it is not a high priority for
me to track down that issue. But if you think those two symbolic links
should do the trick, we can try again to see if the issue is reproducible
and fix it in our next session.

Azad
